File: Examples/Examples/ListView/DataSourceItem.cs
using System;

using Foundation;

namespace Examples
{
	public class DataSourceItem: NSObject
	{
		public DataSourceItem(string name, float value, string group) 
		{
			this.Name = name;
			this.Value = value;
			this.Group = group;
		}

		[Export("Name")]
		public String Name { get; set; }

		[Export("Content")]
		public String Content { get; set; }

		[Export("Value")]
		public nfloat Value { get; set; }

		[Export("Group")]
		public String Group { get; set; }

		[Export("StartDate")]
		public NSDate StartDate { get; set; }

		[Export("EndDate")]
		public NSDate EndDate { get; set; }
	}
}

File: TelerikUIExamplesInSwift/TelerikUIExamplesInSwift/DataSourceItem.swift
//
//  DataSourceItem.swift
//  TelerikUIExamplesInSwift
//
//  Created by Sophia Lazarova on 5/31/16.
//  Copyright © 2016 Telerik. All rights reserved.
//

import UIKit

class DataSourceItem: NSObject {
    var name:String = ""
    var content:String = ""
    var value:Float = 0
    
    init(name: String, value: Float) {
        self.name = name
        self.value = value
    }
    
    var group:String = ""
    
    init(name: String, value: Float, group: String) {
        self.name = name
        self.value = value
        self.group = group
    }
    
    var startDate:NSDate?
    
    var endDate:NSDate?
}

